extra-files/cloud_setup/tool_data_table_conf.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<!--
+    This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
+-->
+<tables>
+    <!-- Locations of all fasta files under genome directory -->
+    <table name="all_fasta" comment_char="#">
+        <columns>value, dbkey, name, path</columns>
+        <file path="/cvmfs/data.galaxyproject.org/byhand/location/all_fasta.loc" />
```

Comment:
Is there any restriction in terms of how the galaxy cvmfs mount can be reached? Is this jetstream-specific?